src/Bridges/ApplicationDI/PresenterFactoryCallback.php

```diff
@@ -44,6 +44,10 @@ public function __invoke(string $class): Nette\Application\IPresenter
 
 		} elseif (!$services) {
 			if ($this->touchToRefresh) {
+				$dir = pathinfo($this->touchToRefresh, PATHINFO_DIRNAME);
```

Comment:
Maybe Nette\Utils\FileSystem::createDir(dirname($this->touchToRefresh)) is be better

Reply:
ok, changed